.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Task
```

Comment:
If this template is for epics then I would name it `Epic` rather than task.

Reply:
it is a template for issue, I mean for task that should be part of epics. I do not know should I change the name to Epic then?

Reply:
then I think the only confusion was that it was set to `title: "[EPIC]: "`. If you change that to `title: "[TASK]: "` then it's all good :D